src/work-allocation-2/containers/all-work-task/all-work-task.component.ts

```diff
@@ -85,7 +85,7 @@ export class AllWorkTaskComponent extends TaskListWrapperComponent {
       };
       return {
         search_parameters: searchParameters,
-        sorting_parameters: [this.getSortParameter()],
+        sorting_parameters: this.getSortParameter(),
```

Comment:
Minor query - can I just have a quick comment on why this change is required please?

Reply:
getSortParameter() returns an array or sortParameter. So this is required

Reply:
Fair enough, was just intrigued as to why it was like this before - LGTM